articles/azure/azs-how-link-storage-explorer.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to link Azure Storage Explorer to Azure Stack | UKCloud Ltd
+description: Provides help for linking Azure Storage Explorer to Azure Stack
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to link Azure Storage Explorer to Azure Stack
+toc_fullpath: Users/How To/Create a virtual machine/azs-how-link-storage-explorer.md
```

Comment:
Don't think the Create a virtual machine bit is needed here

Reply:
fixed